data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+
+mission "Sheragi Archaeology: The Box 8"
+	name "Assembling the Team"
+	description "Take the Hai archaeologists to meet Albert Foster on <destination>."
+	source "Hai-home"
+	destination "Zug"
+	passengers 3
+	cargo "Archaeological Equipment" 1
+	blocked `You arrive at the spaceport, but then you realize your ships don't have enough free bunks or cargo space required to bring the archaeologist team with you. You manage to contact them, and in an embarrassing conversation ask them to wait for you while you make sure you're ready.`
+	
+	to offer
+		has "Sheragi Archaeology: The Box 7b: done"
+	
+	on offer
+		conversation
+			`You meet up with the Hai archaeologists that will accompany you to human space, an all female team. They introduce themselves to you as Eyama, Rakei, and Desyo.`
+			`	Eyama is a linguist specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`
+			`	Rakei is an archaeological engineer, who immediately takes a liking to the cube as soon as she sees it, examining it's structure.`
+			`	Finally, Desyo is a xenoprogramming expert who studies alien software, and seemingly has a lot of experience with Sheragi code. Apparently, the displays of the Sheragi room of the museum all run in this exotic language, unseen by the usual public.`
+			`	After the brief introduction, they each take their equipment and load it on the cargo bay of your ship. Rakei approaches you after everything is in place.`
+			`	"We want to meet the human archaeologist who uncovered the ruins in the Rim. Can you take us there? We will start examining the cube along the way. I think by the time we get there, we'll have something to work with."`
+				accept
+	
+	on complete
+		event "cube decoded" 40 50
+		conversation
+			`It was a long trip, almost halfway across the galaxy. The Hai archaeologists have been friendly to you, giving you updates every time they make a new breakthrough. During this time, the cube, previously thought to be impenetrable, had been disassembled and cleaned. Components that decayed into a mess of oxidized wires and goopy paste over the eons have been identified and removed to substitute for equivalent ones, and a blueprint of the box and its circuits has started to take shape.`
+			`	As soon as you land, you call Albert Foster, who doesn't believe you when you say you got alien archaeologists to help. You then look for the three Hai to figure out how they're going to go unnoticed on a human world. You're surprised to find Eyama wearing long, dark but colorful baggy human clothing with an equally oversized hoodie and a bandanna. You recognize this outfit, very popular with the youth these days, who are generally regarded as delinquents by the previous generation. Additionally, she's carrying two more sets, for her Hai partners as well. It hides her body really well at the cost of making her look like a young boy, and the Hai's relatively short stature doesn't help.`
```

Comment:
`you've gotten` instead of `you got` might sound better.
Also, I'd recommend removing `baggy` from the hair description. Three (arguably four, considering `colorful`) adjectives at once is a little overkill.
`You recognize this outfit, very popular with the youth these days, who are generally regarded as delinquents by the previous generation.`
This seems somehwat unrelated to the rest of the paragraph. You may want to consider either omitting or shortening this sentence.
`It hides her body really well at the cost of making her look like a young boy, and the Hai's relatively short stature doesn't help.`
I'd rephrase this to something closer to: `Although it hides her body well, she looks like a young boy. The Hai's relatively short stature doesn't help.`

Reply:
The outfit description is relevant for the player's response later on, as context for where the idea came to be.